mkdocs/docs/tutorials/knative/canary-progressive.md

```diff
@@ -98,7 +106,7 @@ sed "s+URL_VALUE+${URL_VALUE}+g" $ITER8/samples/knative/canaryprogressive/fortio
           restartPolicy: Never    
     ```
 
-## 3. Create Iter8 experiment
+## 4. Create Iter8 experiment
```

Comment:
When you `look inside` the Iter8-experiment, it still says `--namespace=iter8-system`. The new experiment  should be copied in here...